website/react-magma-docs/src/pages/api/stepper.mdx

```diff
@@ -759,6 +759,279 @@ export function Example() {
 }
 ```
 
+## Orientation
+
+The `orientation` prop allows you to change the orientation of the `Stepper`. By default, the `Stepper` is displayed horizontally unless the `orientation` prop is specified.
+
```

Comment:
We should note here that the vertical stepper only supports `StepperLayout.showLabels` and `StepperLayout.hideLabels`.